Resources/Prototypes/Recipes/Lathes/security.yml

```
@@ -657,3 +657,37 @@
      Steel: 150
      Plastic: 100
      Glass: 20
+
+#LPP
```

yamllint

[warning] missing starting space in comment

(comments)